### ISSUE TITLE
Suggestion - Increasing the villagers's power

### ISSUE THREAD

**Issue by Fathurhoho** [GreyWolfDev/Werewolf#346]
I can not provide data, but in our group (example: @werewolfarena), cultists always win the game.
This is because almost all roles can be converted easily into a cultist especially the villagers.

Usually villager prefers to burn wuffs / sk rather than culstist, so the duration of the game becomes long and often boring. 

Here's my suggestion:
- Give villager the power to reject incitement from the cultist team
- This power can be obtained from the vote/lynch

As an example:
- I voted wuff (sk/cults/etc), and the wuff is burning down.
- My ability increases so it is harder to be instigated by a cultist

Thank you.

**Comment by HellScre4m** [comment]
I have another suggestion,
decrease chance of cult conversion for most roles which their conversion chances are 100% yet, and whenever cult wants to convert them and they miss it, instead of rejecting, ask that player if they want to be a member of cult or not. Else convert them immediately.
Well I'm suggesting this because cult is really overpowered and CH is a hated role for most players.

**Comment by jaimefd** [comment]
I agree cult is overpowered. My idea to fix it would be to add new village roles that can counter cult, while not counter ww or sk that much.

For example, they should add the Faithful as a role (suggested in main role topic, villager that can't be converted into the cult). This would be a unique role that only appears in 11+ games, and isn't guaranteed.

Another idea I had is a role that can protect from cult conversion. I called it the sheriff. Each night you visit someone. That person can't be converted to the cult and you can also know whether that person stayed home or not. (this role can still be converted by cult, it goes back home early, but around 60% chance)

I feel like none of the new roles is fixing this issue, and some even help cult. For example, Oracle can see what a player is NOT. But this is useless in cult games. If you check the mayor to see if it's culted, you will get "player is NOT" and a random vg role in the game. There is no way to check that the player is not a mayor anymore, because there is no mayor in the game anymore.

And the pacifist... well, this is like a ww role to me. Villager that can't lynch and can save a ww from being shot? Damn, they had so much better roles to add. But I guess wolves needed a buff.

**Comment by Olgabrezel** [comment]
I like the original suggestion actually. Let's see if we can do something like that

**Comment by Olfi01** [comment]
hm the original suggestion doesn't make much sense storywise unless you lynched cults though...

**Comment by Olfi01** [comment]
I think #438 states a much better solution to the cult problem.
